src/types.d.ts

```diff
@@ -3080,17 +3095,6 @@ declare namespace bigquery {
   }
 
   namespace routines {
-    /**
-     * Gets the specified routine resource by routine ID.
-     */
-    type IGetParams = {
```

Comment:
@shollyman @tswast so that's interesting :) Is this possibly a breaking change in the API?

Reply:
I'm curious too, is this interface not used anywhere?

Reply:
We never got the OK to implement routines, there's a PR out there for it now but otherwise this never actually got used.